### PR TITLE
002 view job spec

### PR DESCRIPTION
**Pull request by 40175096**

### CODE REVIEW

File: src/main/java/org/kainos/ea/team2/api/JobService.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     */
+    private IJobDAO instanceJobDao;
+
+    /**
+     * create instance of jobservice class using jobdao interface.
+     * @param jobDao
+     */
+    public JobService(final IJobDAO jobDao) {
+        this.instanceJobDao = jobDao;
+    }
```
Comment (eoinmcbennett) [inline src/main/java/org/kainos/ea/team2/api/JobService.java]: Class instance variable should be same name as constructor one

File: src/main/java/org/kainos/ea/team2/exception/FailedToGetException.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package org.kainos.ea.team2.exception;
+
+public class FailedToGetException extends Throwable {
+
```
Comment (eoinmcbennett) [inline src/main/java/org/kainos/ea/team2/exception/FailedToGetException.java:4]: Add a constructor a message here

File: src/main/java/org/kainos/ea/team2/exception/JobDoesNotExistException.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package org.kainos.ea.team2.exception;
+
+public class JobDoesNotExistException extends Throwable {
+}
```
Comment (eoinmcbennett) [inline src/main/java/org/kainos/ea/team2/exception/JobDoesNotExistException.java:17]: Add a constructor a message here

**Review by eoinmcbennett — CHANGES_REQUESTED** [review]

**Review by eoinmcbennett — APPROVED** [review]

File: sqlfiles/003-002-view-job-spec.sql [hunk 4]
```diff
@@ -0,0 +1,56 @@
+USE MovesLikeSwagger_MeganM;
+
+-- add job_specification column attribute to JobRoles table
+ALTER TABLE JobRoles
+	ADD COLUMN job_specification TEXT
+    AFTER job_name;
+
+-- add sharepoint_link column attribute to JobRoles table
+ALTER TABLE JobRoles
+    ADD COLUMN sharepoint_link TEXT
```
Comment (shaunganley) [inline sqlfiles/003-002-view-job-spec.sql:10]: might not need this length of column

File: src/main/java/org/kainos/ea/team2/cli/JobSpecificationResponse.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * Specification summary of job.
+     */
+    private String jobSpecification;
+
+    /**
+     * Link to full job spec on sharepoint.
+     */
+    private String sharepointLink;
+
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/team2/cli/JobSpecificationResponse.java:21]: is there any other info we might want to display on the frontend screen? role name perhaps?

**Review by shaunganley — APPROVED** [review]

**Pull request merged**